core/src/apps/cardano/layout.py

```diff
@@ -181,9 +195,114 @@ async def confirm_certificate(
     await require_confirm(ctx, Paginated(pages))
 
 
+async def confirm_stakepool_parameters(
+    ctx: wire.Context,
+    pool_parameters: CardanoPoolParametersType,
+    network_id: int,
+    protocol_magic: int,
+) -> None:
+    page1 = Text("Confirm transaction", ui.ICON_SEND, ui.GREEN)
+    page1.normal("Confirm:")
+    page1.bold("Stakepool registration")
+    page1.normal("Network:")
+    page1.bold(protocol_magics.to_ui_string(protocol_magic))
+
+    page2 = Text("Confirm transaction", ui.ICON_SEND, ui.GREEN)
+    page2.normal("Pool id: ")
```

Comment:
Why is there is space at the end of the string? (Not only here.)

Reply:
probably remains of shuffling strings around, fixing